src/Microsoft.AspNet.OData.Shared/Query/ODataQueryOptions.cs

```diff
@@ -441,10 +442,11 @@ private static List<string> GetApplySortOptions(ApplyClause apply)
             if (lastTransform.Kind == TransformationNodeKind.Aggregate)
             {
                 var aggregateClause = lastTransform as AggregateTransformationNode;
-                foreach (var expr in aggregateClause.Expressions)
-                {
-                    result.Add(expr.Alias);
-                }
+                Debug.Assert(aggregateClause != null);
```

Comment:
>Debug.Assert [](start = 16, length = 12)

WebAPI uses Contract.Assert.  This should allow you to get rid of the new reference to System.Diagnostics.

Reply:
Thanks for informing me of Contract.Assert! :)